modules/rest-api/pages/rest-set-up-services-existing-nodes.adoc

```diff
@@ -0,0 +1,151 @@
+= Assigning Services on an Existing Node
+
+:description: pass:q[Use the REST API to assign or remove services on an existing node in a cluster, and rebalance the cluster.]
+:page-topic-type: reference
+
+[abstract]
+{description}
+
+[#http-method-and-uri]
+
+=== HTTP Method and URI
+
+----
+POST /controller/rebalance
+----
+
+== Description
+
+Use the following REST API to add or remove non-Data services on the existing nodes in a cluster.
+
```

Comment:
Line 28 needs to be here.  Currently, these two sentences are separated by a NOTE.  It's confusing.  The two sentences go together.

Use the following REST API to add or remove non-Data services on the existing nodes in a cluster.  
Then trigger a rebalance operation.